src/libraries/Directory.Build.targets

```diff
@@ -301,7 +301,7 @@
     <PackageVersion>$(MajorVersion).$(MinorVersion).$(ServicingVersion)</PackageVersion>
   </PropertyGroup>
   
-  <PropertyGroup Condition="Exists('$(PkgProjPath)') and '$(AssemblyVersion)' == '$(MajorVersion).$(MinorVersion).0.0'">       
+  <PropertyGroup Condition="Exists('$(PkgProjPath)') and $(AssemblyVersion.StartsWith('$(MajorVersion).$(MinorVersion)'))">       
```

Comment:
we need to keep the major & minor version check for assemblies where we have pinned to some older versions like 4.0.* rather than using the default 5.0.0.0

Reply:
nit: add a `.` after `$(MinorVersion)` .  In case someone happened to have a version like 3.10.x.x when we were checking for 3.1.x.x